### ISSUE TITLE
Helm charts successor?

### ISSUE THREAD

**Issue by sleepy-manul** [sonatype/nxrm3-helm-repository#32]
I just read that the Helm charts for NXRM are to be decommissioned. What is the successor solution? Will there be new charts or a Kubernetes operator?

**Comment by ilyesFh** [comment]
Could you please explain why you will abandon this Chart and do not recommend it for a resilient onprem installation ?

**Comment by BlueCog** [comment]
Also interested in a answer on this!

**Comment by BlueCog** [comment]
@sleepy-manul They just updated the Readme with more details.

**Comment by mitchellmaler** [comment]
I feel like fully deprecating the helm chart is a bit extreme if the only concern is using the embedded database. 

They should just version bump the chart and make it a requirement to have an external database to connect to along with making it require to setup in the values.

I guess another option is to understand why running in containers could corrupt the local db and see if that could be solved for.

**Comment by ChrisJBurns** [comment]
We are using Nexus Repo OSS (on EC2) with EBS storage and are currently looking at the Helm Charts we need to use in order to install into K8s. We were hoping we could just install into K8s and point it to the EBS Volume, but all I'm seeing is that Postgres is only available? When did the Postgres resource become mandatory? Is EBS not supported now? Also, does this repo not have OSS charts?

**Comment by ChrisJBurns** [comment]
@mitchellmaler I completely agree. The fact that there will not be a way of installing the OSS version of Repo Manager to Kubernetes because of one limitation seems a bit extreme. I'm hoping this decision is re-evaluated soon

**Comment by hybby** [comment]
Also concerned about the support status of this chart - how are OSS versions to be managed and kept up to date if no new versions are published? Eventually, the latest version of the chart will likely become un-runnable in some future Kubernetes version where a resource used by this chart is deprecated and removed.

Please consider re-evaluating this decision.

**Comment by jhaumont** [comment]
hello @bobotimi 
Any answer regarding the previous concern from sonatype?

**Comment by vswaminathan777** [comment]
Deploying Nexus Repository in containers with an embedded database has been known to corrupt the database under some circumstances. We strongly recommend that you use an external PostgreSQL database for Kubernetes deployments.

Below is the link to the Helm charts that can be used for on-prem/AWS/Azure deployments along with an external PostgreSQL database. 
[](https://github.com/sonatype/nxrm3-ha-repository/tree/main/nxrm-ha
)

**Comment by Roberdvs** [comment]
> Deploying Nexus Repository in containers with an embedded database has been known to corrupt the database under some circumstances. We strongly recommend that you use an external PostgreSQL database for Kubernetes deployments.
> 
> Below is the link to the Helm charts that can be used for on-prem/AWS/Azure deployments along with an external PostgreSQL database. [sonatype/nxrm3-ha-repository@`main`/nxrm-ha](https://github.com/sonatype/nxrm3-ha-repository/tree/main/nxrm-ha?rgh-link-date=2024-01-08T13%3A49%3A38Z)

Well, both the usage of an external database and high availability setups are only available with a Nexus Repository Pro license, which makes those charts unusable for the OSS version.

Furthermore, if "deploying Nexus Repository in containers with an embedded database has been known to corrupt the database under some circumstances" and [the documentation highly recommends using an external PostgreSQL database](https://help.sonatype.com/repomanager3/planning-your-implementation/database-options), it is even more disappointing that the option to use an external database is not available for the OSS version because essentially means there's no way to deploy it in Kubernetes safely, even if there are no official charts available.

**Comment by vswaminathan777** [comment]
@Roberdvs We at Sonatype fully understand your concern. Though I cannot promise anything at this point of time, you can expect some exciting updates around our product tiers and supported capabilities within the next few months that could address your concerns. Please stay tuned.
